app/src/main/java/de/xikolo/models/migrate/RealmSchemaMigration.java

```diff
@@ -147,6 +147,10 @@ public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
         if (oldVersion == 10) {
             schema.get("Channel").addRealmObjectField("stageStream", schema.get("VideoStream"));
         }
+
+        if (oldVersion == 11) {
+            schema.get("Item").addRealmObjectField("timeEffort", schema.get("time_effort"));
```

Comment:
Should be `int.class` instead of `schema.get("time_effort")` because the field is an Integer.

Reply:
True, but why did it work? 🤔

Reply:
RealmMagic©

Reply:
done